com.reprezen.swagedit.feature/feature.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="com.reprezen.swagedit.feature"
-      label="SwagEdit"
+      label=" KaiZen-OpenAPI-Editor"
```

Comment:
@tedepstein fixed in https://github.com/RepreZen/KaiZen-OpenAPI-Editor/pull/279/commits/97a54ba50e3a98f6e1e87c5ae542218faf6bb3a6#diff-abd602282327c9fa1b93b03213da3def